repos/system_upgrade/el7toel8/actors/peseventsscanner/actor.py

```diff
@@ -21,7 +21,7 @@ class PesEventsScanner(Actor):
     """
 
     name = 'pes_events_scanner'
-    consumes = (InstalledRedHatSignedRPM, RepositoriesBlacklisted, RepositoriesMap, RpmTransactionTasks)
+    consumes = (InstalledRedHatSignedRPM, RepositoriesExcluded, RepositoriesMap, RpmTransactionTasks)
```

Comment:
@ZhukovGreen this is against our policy. Follow the deprecation process. You can add the RepositoriesExcluded but you have to keep support of RepositoriesBlacklisted and set it as deprecated.

Reply:
Thanks for pointing out. I will adopt the deprecation process. @pirat89

Reply:
@ZhukovGreen I am still missting the `RepositoriesBlacklisted` in the consumed set. Deprecation policy tells, you have to keep the deprecated functionality until removed.

Reply:
I don't think *our* actor needs to consume it. That's not how I understand deprecation. If we deprecate a model, it's definition still needs to exist (of course), we still need to produce the messages of this model. Nothing more.

Reply:
@bocekm they have to, otherwise it is not deprecated model and we could remove it.

Reply:
You're right. If any third party actor produces the message through which they want to blacklist a repository and if we stop consuming/processing the message, we would stop excluding such their repository.